pyccl/halos/concentration.py

```diff
@@ -378,7 +377,7 @@ class ConcentrationIshiyama21(Concentration):
     def __init__(self, mdef=None, relaxed=False, Vmax=False):
         self.relaxed = relaxed
         self.Vmax = Vmax
-        super(ConcentrationIshiyama21, self).__init__(mass_def=mdef)
+        super().__init__(mass_def=mdef)
```

Comment:
why this change?

Reply:
`super` is smart in python 3, this was only required in python 2, so just cleaning up the code.